Python/challenges/day_2.py

```diff
@@ -0,0 +1,23 @@
+def get_result(round, strategy=False):
+    score_matrix = [3, 6, 0]
+    moves = round.split(" ")
+    opponent = int(ord(moves[0]) - 65)
+    you = int(ord(moves[1]) - 88)
```

Comment:
When working with `ord()`, subtracting by 65 may be confusing. Instead, consider replacing it with `ord('A')`.

Reply:
```suggestion
    opponent = int(ord(moves[0]) - ord("A"))
    you = int(ord(moves[1]) - ord("X"))
```
this does look at lot cleaner, good suggestion